hummingbot/strategy/pure_market_making/pure_market_making.pyx

```diff
@@ -762,6 +852,23 @@ cdef class PureMarketMakingStrategy(StrategyBase):
         finally:
             self._last_timestamp = timestamp
 
+    def get_order_amount(self, reference_price: Decimal) -> Decimal:
+        order_amount = self._order_amount
+        if self._quote_order_amount > 0:
+            order_amount = self._quote_order_amount / reference_price
+        return order_amount.quantize(Decimal('1e-4'), rounding=ROUND_DOWN)
+        
+
+    def get_order_level_amount(self, reference_price: Decimal) -> tuple[Decimal, Decimal]: 
+            buy_order_level_amount = self._buy_level_amount
+            sell_order_level_amount = self._sell_level_amount
```

Comment:
```suggestion
        buy_order_level_amount = self._buy_level_amount
        sell_order_level_amount = self._sell_level_amount
```

Reply:
The indentation here is not correct, compiling is failing for me.

Reply:
Fixed